_README.md

```diff
@@ -9,7 +9,7 @@ The steps below will walk you through setting up your own instance of the projec
 ### Install Project Dependencies
 To set up the development environment for this website, you'll need to install the following on your system:
 
-- [Node](http://nodejs.org/) v10 (To manage multiple node versions we recommend [nvm](https://github.com/creationix/nvm))
+- [Node](http://nodejs.org/) v14 (To manage multiple node versions we recommend [nvm](https://github.com/creationix/nvm))
```

Comment:
I would suggest avoiding adding the node version to the readme as it might change and became outdated. Maybe a link to `.nvmrc`?